nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/stream/StreamProcessorDescriptor.java

```diff
@@ -205,6 +208,10 @@ public ComputationPolicyBuilder createPolicyBuilder() {
     @XNode("@defaultCodec")
     public String defaultCodec;
 
+    // @since 11.1
+    @XNode("@defaultExternal")
+    public boolean defaultExternal;
```

Comment:
What is this for? Please add some javadoc/explanations and maybe test

Reply:
Like other @default* props, it sets a default value for all streams/computation of the processor. 
Here all stream that don't override the property explicitly will have this default external value.
I will add a javadoc